src/scenic/core/regions.py

```diff
@@ -1841,7 +1905,7 @@ def __init__(self, points=None, polylines=None, name=None):
                 # Update last point
                 last_pt = cast_pt
 
-        self.vert_to_vec = {val:key for key,val in self.vec_to_vert.items()}
+        self.vert_to_vec = tuple(key for key,val in self.vec_to_vert.items())
```

Comment:
```suggestion
        self.vert_to_vec = tuple(self.vec_to_vert)
```
(or stick in a `.keys()` if you want to be more explicit)